### PR TITLE
Support Elm 0.19

### PR DESCRIPTION
**Pull request by Punie**
Hi !

I made a small change to support Elm 0.19 (by locating `elm.json` instead of `elm-package.json`).
As far as I know, other than this filename change, there were no fundamental differences on how modules and imports are resolved in 0.19.

This upgrade is necessary for [`node-elm-compiler`](https://github.com/rtfeldman/node-elm-compiler) and [`elm-webpack-loader`](https://github.com/elm-community/elm-webpack-loader) to support Elm 0.19 [(PR #142 on elm-webpack-loader)](https://github.com/elm-community/elm-webpack-loader/pull/142#issuecomment-414904042)

### CODE REVIEW

**Comment by hecrj** [comment]
@Punie I have opened a PR in your fork that also fixes the tests for this branch: https://github.com/Punie/find-elm-dependencies/pull/1

**Comment by Punie** [comment]
@hecrj Thank you ! I merged it just now :)

File: index.js [hunk 1]
```diff
@@ -71,14 +71,14 @@ function findAllDependencies(file, knownDependencies, sourceDirectories, knownFi
...
-// elm-package.json file that includes it and get all its source directories.
+// elm.json file that includes it and get all its source directories.
 function getElmPackageSourceDirectories(baseDir, currentDir) {
   if (typeof currentDir === "undefined") {
     baseDir = path.resolve(baseDir);
     currentDir = baseDir;
   }
 
-  var elmPackagePath = path.join(currentDir, 'elm-package.json');
+  var elmPackagePath = path.join(currentDir, 'elm.json');
```
Comment (halfzebra) [inline index.js:81]: Not really `elmPackagePath` anymore, isn't it more like `elmJsonPath`?

**Comment by eeue56** [comment]
Looks good to me. @stoeffel / @rtfeldman can you merge and release?

File: index.js (hunk 1 above)
Comment (Punie) [reply index.js:81]: I guess it's still the file that defines the packages on which the elm build depends.
I'm happy to make the change to `elmJsonPath` if this terminology seems more sensible :)

**Comment by rtfeldman** [comment]
Done - thanks @Punie!

https://www.npmjs.com/package/find-elm-dependencies/v/2.0.0

**Pull request merged**